manage_commands.py

```diff
@@ -0,0 +1,177 @@
+import json
+
+from app import db
+from app.data import initial_data_carriers
+from flask_script import Command
+from sqlalchemy.exc import IntegrityError
+
+
+def save_models(elements, model_class):
+    for json_element in elements:
+        model = model_class()
+        for k, v in json_element.items():
+            setattr(model, k, v)
+        db.session.add(model)
+        try:
+            db.session.commit()
+        except IntegrityError:
+            db.session.rollback()
+
+
+class Populate(Command):
+    def run(self):
+        populate()
+
+
+class PopulateAntennas(Command):
```

Comment:
This method has changed in the original project (and the carrier-antenna link). Maybe it would be a good decision to see the changes before resolve the pull request.